### ISSUE TITLE
Enable Vulkan on beta channel

### ISSUE THREAD

**Issue by SergeyZhukovsky** [brave/brave-browser#24933]

**Comment by kjozwiak** [comment]
@Uni-verse @srirambv once https://github.com/brave/brave-variations/pull/334 is merged, we'll basically just need to go through the following STR/Cases:

* launch `1.44.x` (BETA channel)
* you'll notice that `Vulkan` is enabled via `brave://gpu` which is expected
* restart the browser and you should see `Vulkan` disabled via `brave://gpu` and `VulkanStudy:Enabled` via `brave://version`

Some background why `Vulkan` is initially enabled can be found via https://github.com/brave/brave-variations/pull/333#issuecomment-1226729431.

Because we have `Vulkan` disabled via Griffin, the current behaviour should be the following on `production` before the above gets merged into `production`:

> * Current Behaviour:  `Vulkan` enabled by default on first launch but when re-launching the browser, `Vulkan` will be disabled]
>   * can easily tell my checking `brave://version` after restarting and seeing `VulkanStudy:Disabled`
> * Once above is merged: `Vulkan` is enabled by default but will stay enabled after restarting the browser
>   * can easily tell my checking `brave://version` after restarting and seeing `VulkanStudy:Enabled`

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Pixel 6` running `Android 13` using the following build(s):

```
Brave | 1.44.95 Chromium: 106.0.5249.40 (Official Build) (32-bit)
--- | ---
Revision | fab1d91915d2722d6339aaa7f4e9ce44f1e9b103-refs/branch-heads/5249@{#442}
OS | Android 13; Build/T1B1.220819.006
```

Went through the STR/Cases outlined above and ensured that `VulkanStudy:Enabled` on `production` without the need of using any flags. Also ensured that `Vulkan` is `Enabled` via `brave://gpu`.

**`First launch (Girffin studies not pulled yet)`**

**`Example`** | **`Example`**
----------------|---------------
![Screenshot_20220921-161513](https://user-images.githubusercontent.com/2602313/191602546-ea818740-c16f-4825-80a8-e3978a996ee5.png) | ![Screenshot_20220921-161526](https://user-images.githubusercontent.com/2602313/191602568-ee87f734-22cb-4059-8ed6-829545747512.png)

**`Second launch (Girffin studies pulled from production)`**

**`Example`** | **`Example`**
----------------|---------------
![Screenshot_20220921-161543](https://user-images.githubusercontent.com/2602313/191602670-638065d4-24ed-4923-b787-b77f3a02236c.png) | ![Screenshot_20220921-161550](https://user-images.githubusercontent.com/2602313/191602679-19a703bb-b231-432d-9e24-5c556ea820b2.png)

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Samsung Tab S8 Ultra` running `Android 12` using the following build(s):

```
Brave | 1.44.95 Chromium: 106.0.5249.40 (Official Build) (32-bit)
--- | ---
Revision | fab1d91915d2722d6339aaa7f4e9ce44f1e9b103-refs/branch-heads/5249@{#442}
OS | Android 12; Build/SP2A.220305.013
```

Went through the STR/Cases outlined above and ensured that `VulkanStudy:Enabled` on `production` without the need of using any flags. With the `Samsung Tab S8 Ultra`, I basically ensured that https://github.com/brave/brave-browser/issues/25124 wasn't occuring.

Example of `VulkanStudy:Disabled` and the rendering issue occurring:

**`Example`** | **`Example`**
---------------|------------------
![Screenshot_20220921_200841_Brave](https://user-images.githubusercontent.com/2602313/191631555-8c91155e-8156-43b8-b026-4bcac4eb8dae.jpg) | ![Screenshot_20220921_200904_Brave](https://user-images.githubusercontent.com/2602313/191631565-1d0ed087-2e42-44e2-a32a-8343c869b3f0.jpg)

Notice that the tabs are being correctly rendered without any graphical artifacts.

Example of `VulkanStudy:Enabled` and the rendering issue occurring:

**`Example`** | **`Example`** | **`Example`**
---------------|----------------|-----------------
![Screenshot_20220921_201107_Brave](https://user-images.githubusercontent.com/2602313/191631727-bc7ea856-89f5-4761-8785-5941b33cd15c.jpg) | ![Screenshot_20220921_201053_Brave](https://user-images.githubusercontent.com/2602313/191631733-9f6ae036-4a9b-4836-b66a-9a618f3423c2.jpg) | ![Screenshot_20220921_201143_Brave](https://user-images.githubusercontent.com/2602313/191631744-a8681965-4b77-457e-9916-3ddc07ccd807.jpg)

**Comment by kjozwiak** [comment]
@Uni-verse @srirambv @stephendonner can you guys check the following on your tablets and report back what `brave://gpu` reports?

* install and launch `1.44.95 Chromium: 106.0.5249.40`
* restart the browser and check `brave://version`
* ensure that you landed in the `VulkanStudy:Enabled` cohort (currently set as 50%)
* once you've been placed into the `VulkanStudy:Enabled` cohort, check if `Vulkan` appears enabled via `brave://gpu`

**Comment by btlechowski** [comment]
Verification passed on Brave v1.44.97  on Samsung Galaxy Tab S5e (Android 11.0)

Verified test plan from https://github.com/brave/brave-browser/issues/24933#issuecomment-1254358224

Verified `Vulkan` is enabled

![Screenshot_20220922-163012_Brave](https://user-images.githubusercontent.com/34715963/191776681-a7413977-02a1-454e-80c8-06282f442541.jpg)|![Screenshot_20220922-162622_Brave](https://user-images.githubusercontent.com/34715963/191776685-4ea2171f-3d04-4791-a5ae-32273fc04ec0.jpg)
--|--
